### DISCUSSION TITLE
Has anyone tried uploading a widget plugin for v2?

### DISCUSSION THREAD

**Discussion by Betanoir** [medusajs/medusa#10088] [Q&A]
I've got a plugin ready to be released and I'm not entirely sure when the "publishing a plugin" docs are going to be released.

If you have successfully released a widget plugin for v2, please let me know 🙂

**Comment by SalahAdDin** [comment]
@Betanoir did you check the documentation?

**Reply by Betanoir** [reply]
Yes, thoroughly

By the looks of things, plugins aren't supported yet so I'm going to have to wait till the devs release it
